### ISSUE TITLE
No meta data or artworks when queue repeats 

### ISSUE THREAD

**Issue by erkr** [music-assistant/hass-music-assistant#518]
### What version of Music Assistant has the issue?

2022.6.6

### The problem

Once the queue repeats, the players don’t show art works or meta data anymore. Playback continues as expected.

### How to reproduce

Wait till queue is completed the first round

### Additional information

The player state:
``` 
volume_level: 0.44
is_volume_muted: false
media_content_type: music
media_position: 0
media_position_updated_at: '2022-06-26T12:59:59.234576+00:00'
app_id: mass
app_name: Music Assistant
shuffle: true
repeat: all
entity_picture_local: null
source_entity_id: media_player.everywhere_57
is_group: true
group_childs:
  - media_player.audio57serre
  - media_player.audio57woonkmr
  - media_player.audio57keuken
group_parents: []
active_queue: media_player.everywhere_57
items_in_queue: 50
device_class: speaker
friendly_name: Everywhere_57
supported_features: 456639
``` 

this is how the queue and player look like in MA:
![7BB5882C-4716-43A7-AA9E-88ABC97C437A](https://user-images.githubusercontent.com/4363599/175815608-cba426f6-22ea-4e10-a70e-cf1318e9576c.png)

### What version of Home Assistant Core are your running

2022.6.6

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Raspberry Pi

**Comment by erkr** [comment]
@marcelveldt 
Include this one for 2022.7.0?

**Comment by marcelveldt** [comment]
No, 2022.7.x is the next major release. This is a bugfix I pickup when doing a bugrelease.

**Comment by erkr** [comment]
Even better 😎

**Comment by marcelveldt** [comment]
should be fixed in 2022.6.7, if not please reopen, thanks!
